sqflite_common/lib/sqlite_api.dart

```diff
@@ -454,9 +451,21 @@ abstract class Batch {
   /// During [Database.onCreate], [Database.onUpgrade], [Database.onDowngrade]
   /// (we are already in a transaction) or if the batch was created in a
   /// transaction it will only be commited when
-  /// the transaction is commited ([exclusive] is not used then)
-  Future<List<Object?>> commit(
-      {bool? exclusive, bool? noResult, bool? continueOnError});
+  /// the transaction is commited ([exclusive] is not used then).
+  ///
+  /// Otherwise, sqflite will start a transaction for this batch if
```

Comment:
Good job on the doc here thanks. Maybe a good doc is enough to justify any choice of parameter name!